packages/node_modules/cerebral/src/views/View.js

```diff
@@ -220,7 +220,6 @@ class View {
   /*
     Go through dependencies and identify state trackers and
     merge in their state dependencies
-    TODO: don't need props
   */
   getDependencyTrackersDependencyMaps(props) {
```

Comment:
@christianalfoni we don't use props argument inside the function. Can I remove it?